qemu/tests/virtio_fs_share_data.py

```diff
@@ -1016,40 +1016,43 @@ def get_vfsd_num():
                                 vfsd_num += len(vfsd_ps.strip().splitlines())
                         return vfsd_num
 
-                    error_context.context(
-                        "Check virtiofs daemon before reboot vm.", test.log.info
-                    )
-                    vfsd_num_bf = get_vfsd_num()
-                    error_context.context(
-                        "Reboot guest and check virtiofs daemon.", test.log.info
-                    )
-                    session = vm.reboot(session)
-                    if not vm.is_alive():
-                        test.fail("After rebooting vm quit unexpectedly.")
-                    vfsd_num_af = get_vfsd_num()
-                    if vfsd_num_bf != vfsd_num_af:
-                        test.fail(
-                            "Virtiofs daemon is different before and after reboot.\n"
-                            "Before reboot: %s\n"
-                            "After reboot: %s\n",
-                            (vfsd_num_bf, vfsd_num_af),
+                    for reboot_cmd in ["shell", "system_reset"]:
+                        error_context.context(
```

Comment:
As there are two reboot operations, for the second reboot please re-mount the viofs file system.

Reply:
Thank you for review. Updated. @xiagao